src/modules/session/slice.ts

```diff
@@ -19,14 +18,10 @@ const sessionSlice = createSlice({
     setSessionErrorMessage(state, action: PayloadAction<string>) {
       state.errorMessage = action.payload;
     },
+    logOut: handleLogout,
```

Comment:
Is the state automatically getting passed into handleLogout? It could be written as 
`
    logOut(state) {
      handleLogout(state);
    },
`  correct? Just curious

Reply:
Yeah, that should be equivilent. It that notation, your more explicity calling the function, and in the current notatin, your just setting that object value to the function.

Similarly, `setSessionErrorMessage` could also be written as 

```
setSessionErrorMessage: (state, action: PayloadAction<string>) => {
  state.errorMessage = action.payload;
},
```